python/ray/autoscaler/_private/kuberay/run_autoscaler.py

```diff
@@ -17,7 +17,21 @@
 
 
 def run_kuberay_autoscaler(cluster_name: str, cluster_namespace: str):
-    """Wait until the Ray head container is ready. Then start the autoscaler."""
+    """Wait until the Ray head container is ready. Then start the autoscaler.
+
+    For kuberay's autoscaler integration, the autoscaler runs in a sidecar container
```

Comment:
can we just move this to line 44? Docstring should explain the API (`run_kuberay_autoscaler `), but this string looks like an implementation detail

Reply:
This has been moved to where `_setup_logging()` is called further down.